frontend/src/Authentication.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Component } from 'react';
+import authService from './services/authService'
+import './Authentication.css'
+
+class Authentication extends Component {
+
+    state = {
+        username: "",
+        email: "",
+        password1: "",
+        password2: "",
+        register: false
+    };
+
+    validateLoginForm() {
+        return this.state.username.length > 0 && this.state.password1.length > 0;
+    }
+    validateRegisterForm() {
+        return this.state.username.length > 0 && this.state.email.length > 0 && this.state.password1.length > 0 && this.state.password2.length > 0;
+    }
+    handleSubmit(event) {
+        event.preventDefault();
+    }
+
+    loginView() {
+        return (
```

Comment:
Move these two to another files like /components/loginComponent.js, and /components/registerComponent.js
They don't have to be in class, can be just functional components.